[2022-01-31 15:07:48,335] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:07:48,341] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:07:48,341] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:07:48,341] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-31 15:07:48,341] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:07:48,349] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-25T09:00:00+00:00
[2022-01-31 15:07:48,351] {standard_task_runner.py:54} INFO - Started process 5769 to run task
[2022-01-31 15:07:48,386] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-25T09:00:00+00:00', '--job_id', '3', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpkwry9d9d']
[2022-01-31 15:07:48,387] {standard_task_runner.py:78} INFO - Job 3: Subtask twitter_aluraonline
[2022-01-31 15:07:48,402] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [running]> on host 1a222f561522
[2022-01-31 15:07:48,417] {taskinstance.py:1150} ERROR - The conn_id `twitter_default` isn't defined
Traceback (most recent call last):
  File "/run/datapipeline/.env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/run/datapipeline/airflow/plugins/operators/twitter_operator.py", line 52, in execute
    for pg in hook.run():
  File "/run/datapipeline/airflow/plugins/hooks/twitter_hook.py", line 52, in run
    session = self.get_conn()
  File "/run/datapipeline/.env/lib/python3.8/site-packages/airflow/hooks/http_hook.py", line 62, in get_conn
    conn = self.get_connection(self.http_conn_id)
  File "/run/datapipeline/.env/lib/python3.8/site-packages/airflow/hooks/base_hook.py", line 87, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/run/datapipeline/.env/lib/python3.8/site-packages/airflow/hooks/base_hook.py", line 83, in get_connections
    return secrets.get_connections(conn_id)
  File "/run/datapipeline/.env/lib/python3.8/site-packages/airflow/secrets/__init__.py", line 59, in get_connections
    raise AirflowException("The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `twitter_default` isn't defined
[2022-01-31 15:07:48,418] {taskinstance.py:1187} INFO - Marking task as FAILED. dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220125T090000, start_date=20220131T150748, end_date=20220131T150748
[2022-01-31 15:07:53,330] {local_task_job.py:102} INFO - Task exited with return code 1
[2022-01-31 15:18:14,215] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:18:14,220] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:18:14,220] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:18:14,220] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-31 15:18:14,220] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:18:14,225] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-25T09:00:00+00:00
[2022-01-31 15:18:14,227] {standard_task_runner.py:54} INFO - Started process 15754 to run task
[2022-01-31 15:18:14,262] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-25T09:00:00+00:00', '--job_id', '8', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpi2dzrz1u']
[2022-01-31 15:18:14,263] {standard_task_runner.py:78} INFO - Job 8: Subtask twitter_aluraonline
[2022-01-31 15:18:14,281] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [running]> on host 1a222f561522
[2022-01-31 15:18:14,311] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-01-31 15:18:14,312] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z
[2022-01-31 15:18:14,344] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:14,633] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxttnvxl7hjlamdenappqjbst
[2022-01-31 15:18:14,634] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:15,021] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxt82zharpkjvvr4oshj6jxj1
[2022-01-31 15:18:15,022] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:15,313] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvpzra0j43ggkp0mep67su73x
[2022-01-31 15:18:15,314] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:15,583] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvosol4axuho4funj9s6lqvst
[2022-01-31 15:18:15,583] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:15,896] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4789aamvlf8nt0kpr4gixhb15z1
[2022-01-31 15:18:15,897] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:16,172] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47899ovxxkmjc7k5348oqwg714t
[2022-01-31 15:18:16,173] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:18:16,430] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220125T090000, start_date=20220131T151814, end_date=20220131T151816
[2022-01-31 15:18:19,213] {local_task_job.py:102} INFO - Task exited with return code 0
[2022-01-31 15:25:18,389] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:25:18,394] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:25:18,394] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:25:18,394] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-31 15:25:18,394] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:25:18,401] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-25T09:00:00+00:00
[2022-01-31 15:25:18,403] {standard_task_runner.py:54} INFO - Started process 23196 to run task
[2022-01-31 15:25:18,437] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-25T09:00:00+00:00', '--job_id', '9', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpbfjapfbg']
[2022-01-31 15:25:18,437] {standard_task_runner.py:78} INFO - Job 9: Subtask twitter_aluraonline
[2022-01-31 15:25:18,453] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [running]> on host 1a222f561522
[2022-01-31 15:25:18,485] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-01-31 15:25:18,486] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z
[2022-01-31 15:25:18,505] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:18,766] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxttnvxl7hjlamdenappqjbst
[2022-01-31 15:25:18,767] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:19,042] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxt82zharpkjvvr4oshj6jxj1
[2022-01-31 15:25:19,042] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:19,366] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvpzra0j43ggkp0mep67su73x
[2022-01-31 15:25:19,366] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:19,642] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvosol4axuho4funj9s6lqvst
[2022-01-31 15:25:19,642] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:19,945] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4789aamvlf8nt0kpr4gixhb15z1
[2022-01-31 15:25:19,946] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:20,227] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47899ovxxkmjc7k5348oqwg714t
[2022-01-31 15:25:20,228] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:25:20,510] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220125T090000, start_date=20220131T152518, end_date=20220131T152520
[2022-01-31 15:25:23,381] {local_task_job.py:102} INFO - Task exited with return code 0
[2022-01-31 15:37:32,225] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:37:32,229] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:37:32,229] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:37:32,230] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-31 15:37:32,230] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:37:32,236] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-25T09:00:00+00:00
[2022-01-31 15:37:32,238] {standard_task_runner.py:54} INFO - Started process 3023 to run task
[2022-01-31 15:37:32,267] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-25T09:00:00+00:00', '--job_id', '10', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpur5qvcnw']
[2022-01-31 15:37:32,268] {standard_task_runner.py:78} INFO - Job 10: Subtask twitter_aluraonline
[2022-01-31 15:37:32,283] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [running]> on host 1a222f561522
[2022-01-31 15:37:32,313] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-01-31 15:37:32,314] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z
[2022-01-31 15:37:32,333] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:32,622] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxttnvxl7hjlamdenappqjbst
[2022-01-31 15:37:32,622] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:32,939] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxt82zharpkjvvr4oshj6jxj1
[2022-01-31 15:37:32,940] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:33,218] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvpzra0j43ggkp0mep67su73x
[2022-01-31 15:37:33,219] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:33,524] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvosol4axuho4funj9s6lqvst
[2022-01-31 15:37:33,525] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:33,820] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4789aamvlf8nt0kpr4gixhb15z1
[2022-01-31 15:37:33,820] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:34,129] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47899ovxxkmjc7k5348oqwg714t
[2022-01-31 15:37:34,130] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:37:34,371] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220125T090000, start_date=20220131T153732, end_date=20220131T153734
[2022-01-31 15:37:37,223] {local_task_job.py:102} INFO - Task exited with return code 0
[2022-01-31 15:41:20,913] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:41:20,918] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [queued]>
[2022-01-31 15:41:20,918] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:41:20,918] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-31 15:41:20,918] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-31 15:41:20,924] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-01-25T09:00:00+00:00
[2022-01-31 15:41:20,926] {standard_task_runner.py:54} INFO - Started process 6546 to run task
[2022-01-31 15:41:20,955] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-01-25T09:00:00+00:00', '--job_id', '11', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpxe381vw8']
[2022-01-31 15:41:20,956] {standard_task_runner.py:78} INFO - Job 11: Subtask twitter_aluraonline
[2022-01-31 15:41:20,970] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-01-25T09:00:00+00:00 [running]> on host 1a222f561522
[2022-01-31 15:41:21,001] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-01-31 15:41:21,003] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z
[2022-01-31 15:41:21,032] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:21,323] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxttnvxl7hjlamdenappqjbst
[2022-01-31 15:41:21,323] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:21,599] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mxt82zharpkjvvr4oshj6jxj1
[2022-01-31 15:41:21,599] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:21,941] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvpzra0j43ggkp0mep67su73x
[2022-01-31 15:41:21,942] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:22,246] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47mvosol4axuho4funj9s6lqvst
[2022-01-31 15:41:22,246] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:22,562] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe4789aamvlf8nt0kpr4gixhb15z1
[2022-01-31 15:41:22,562] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:22,908] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-01-25T09:00:00.00Z&end_time=2022-01-26T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe47899ovxxkmjc7k5348oqwg714t
[2022-01-31 15:41:22,908] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-01-31 15:41:23,175] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220125T090000, start_date=20220131T154120, end_date=20220131T154123
[2022-01-31 15:41:25,911] {local_task_job.py:102} INFO - Task exited with return code 0
